src/usr/local/buildpack/util.sh

```diff
@@ -190,6 +190,12 @@ function require_distro () {
   esac
 }
 
+function get_distro() {
+  require_distro
+  # shellcheck source=/dev/null
+  . /etc/os-release && echo "${VERSION_CODENAME}"
```

Comment:
I think we need a subshell here, otherwise we get all variables in our scope.
```suggestion
  echo $(. /etc/os-release && echo "${VERSION_CODENAME}")
```